frontend/src/static/css/style.scss

```diff
@@ -183,6 +184,11 @@ header{
         &--tooltip{
             color:mui-color("grey");
         }
+
+        &--octocat{
```

Comment:
instead of octocat, probably can rename to repourl-button or something.
do change the code view as well so that they're standardized.

Reply:
Can I know what is the meaning of "change the code view as well so that they're standardized"?

Reply:
Do you mean that adding an icon at the title of code view page?

Reply:
Hi @fzdy1914 

> change the code view as well so that they're standardized

Yes if possible, I think it would be better to contain the code view link within the icon itself as well for standardization. Then we can probably change the clicking of title to hide the ramp chart or something in the future. @damithc your opinion on this?